user/yaml/README.md

```diff
@@ -0,0 +1,327 @@
+ placement translator is running (log file: /kubestellar/kubestellar-logs/placement-translator-log.txt)
+****************************************
+Finished launching KubeStellar ...
+****************************************
+Current workspace is "root".
+< Create secrets >-----------------------------------------
+Ensure secret in the curent namespace...
+secret/kubestellar created
+Ensure secret in namespace "default"...
+secret "kubestellar" deleted
+secret/kubestellar created
+Ready!
+```
+
+Note that alternatively, one can wait for the `kubestellar` secret to be created.
+
+```shell
+kubectl wait -n kubestellar \
+  --for=condition=ready pod \
+  --selector=app=kubestellar-server \
+  --timeout=120s
+
+until kubectl logs -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) 2>&1 | grep -Fxq "Ready!"
+do
+  sleep 1
+done
+```
+
+After the deployment has completed, **KubeStellar** `admin.kubeconfig` can be in three ways:
+
+- the `kubestellar` secret in the `kubestellar` namespace or any other namespace listed in `SECRET_NAMESPACES` environment variable;
+- directly from the `kubestellar` pod in the `kubestellar` namespace at the location `/home/kubestellar/.kcp/external.kubeconfig`.
+
+## Deploy **KubeStellar** in an **OpenShift** cluster
+
+In this case use the following YAML to deploy  **KubeStellar** `stable` in a `kubestellar` namespace:
+
+```shell
+kubectl apply -f kubestellar-server-route.yaml
+```
+
+Then follow the same instructions for the Kind cluster deployment.
+
+## Access **KubeStellar** service directly from the host OS without `admin.kubeconfig` or plugins
+
+Since **kubectl**, **kcp** plugins, and **KubeStellar** executables are include in the **KubeStellar** container image we can operate KubeStellar directly from the host OS using `kubectl`, for example:
+
+```shell
+$ kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) -- kubectl ws tree
+.
+└── root
+    ├── compute
+    └── espw
+
+$ kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) -- kubectl ws create imw
+Workspace "imw" (type root:organization) created. Waiting for it to be ready...
+Workspace "imw" (type root:organization) is ready to use.
+```
+
+## Access **KubeStellar** service from the host OS by extracting the `admin.kubeconfig` and the plugins from the pod
+
+In this case the host OS will need a copy of **kcp** plugins, and **KubeStellar** plugins, assuming that the architecture of the client is the same of the container, the executables can be extracted from the container:
+
+```shell
+kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) -- tar cf - "/home/kubestellar/kcp-plugins" | tar xf - --strip-components=2

```

Comment:
Could this be simpler with a `-C /home/kubestellar` on the tarchive creation command, saving the need to `--strip-components` on the tarchive extraction command?

Reply:
I don't think it is the same. I'm using `strip-components` to remove `/home/bubestellar` from the filenames in the tar.